web/packages/core/src/internal/player/inner.tsx

```diff
@@ -229,6 +229,14 @@ export class InnerPlayer {
             "input",
             this.virtualKeyboardInput.bind(this),
         );
```

Comment:
A couple dozen lines up we may need this for Windows IME, right near `pointerMoveMaxDistance`:

```ts
    private lastCompositionEndTime = 0
```